lib/ts/recipe/thirdpartyemailpassword/index.ts

```diff
@@ -23,6 +23,18 @@ export default class RecipeWrapper {
         return Recipe.init(config);
     }
 
+    static signOut(input?: { userContext?: any }) {
+        return Recipe.getInstanceOrThrow().signOut({
+            userContext: getNormalisedUserContext(input?.userContext),
```

Comment:
Please make sure the tsconfig is the exact same as that of supertokens-website. So things like using "?" won't work

Reply:
Moved as a point in the checklist here: https://github.com/supertokens/for-zenhub/issues/28